website/docs/r/kusto_cluster.html.markdown

```diff
@@ -0,0 +1,80 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_kusto_cluster"
+sidebar_current: "docs-azurerm-resource-kusto-cluster"
+description: |-
+  Manages Kusto / Data Explorer Cluster
+---
+
+# azurerm_kusto_cluster
+
+Manages a Kusto (also known as Azure Data Explorer) Cluster
+
+## Example Usage
+
+This example provisions a basic Kusto Cluster.
```

Comment:
since there's only a single example we tend to remove these:

```suggestion
```

Reply:
Fixed